xarray/computation/rolling.py

```diff
@@ -168,7 +168,7 @@ def _reduce_method(  # type: ignore[misc]
 
         bottleneck_move_func = getattr(bottleneck, "move_" + name, None)
         if module_available("numbagg"):
-            import numbagg
+            import numbagg  # type: ignore[import-not-found, unused-ignore]
```

Comment:
This was introduced to appease the typechecking on the `upstream-dev-ci` action. How is this action used in xarray? Just for reporting, or are you actively fixing errors in the code using this? (if the former, perhaps I should revert typing changes related to upstream CI)